extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilder.i18n.php

```diff
@@ -3,7 +3,7 @@
 
 $messages['en'] = array(
 	'portableinfoboxbuilder' => 'Infobox Builder',
```

Comment:
you need to change this one to 'infoboxbuilder' for the page to be displayed correctly on Special:SpecialPages